packages/rocketchat-webdav/startup/messageBoxActions.js

```diff
@@ -0,0 +1,60 @@
+/* globals modal, RocketChat */
+import { Tracker } from 'meteor/tracker';
+Tracker.autorun(() => {
```

Comment:
what about moving this to a "better" file? it seems hidden here on a file called `messageBoxActions.js`